packages/devtools_app/lib/src/screens/memory/panes/allocation_profile/allocation_profile_table_view.dart

```diff
@@ -265,12 +269,28 @@ class _AllocationProfileTable extends StatelessWidget {
     _FieldExternalSizeColumn(heap: _HeapGeneration.oldSpace),
   ];
 
-  final AllocationProfileTableViewController allocationProfileController;
+  final AllocationProfileTableViewController controller;
+
+  @override
+  State<_AllocationProfileTable> createState() =>
+      _AllocationProfileTableState();
+}
+
+class _AllocationProfileTableState extends State<_AllocationProfileTable> {
+  late SortDirection sortDirection;
+  late ColumnData<ClassHeapStats> sortColumn;
```

Comment:
Should these two fields be private?

Reply:
They definitely can be, I'm just a little bit looser on naming for members of private classes I guess... :-) Done.